src/sentry/issues/ingest.py

```diff
@@ -237,6 +238,15 @@ def save_issue_from_occurrence(
                         "sdk": normalized_sdk_tag_from_event(event),
                     },
                 )
+        if is_new and occurrence.assignee:
```

Comment:
is it worth quickly making an option to deploy this?

Reply:
I was thinking of putting the option on the crons side so we can stop sending them. This code is pretty well enclosed in try/catch so I'm not too worried about it

Reply:
Although maybe it'd be wise to enclose `assign` in a try/catch too...